packages/datatrak-web/src/api/mutations/useSubmitSurveyResponse.ts

```diff
@@ -51,17 +51,17 @@ export const useSubmitSurvey = () => {
         return;
       }
 
-      return post('submitSurvey', {
+      return post('submitSurveyResponse', {
         data: { ...surveyResponseData, answers },
       });
     },
     {
       onMutate: () => {
         // Send off survey submissions by survey, project, country, and userId
-        gaEvent('submit_survey', params.surveyCode!, survey?.name);
-        gaEvent('submit_survey_by_project', user.project?.code!);
-        gaEvent('submit_survey_by_country', params.countryCode!);
-        gaEvent('submit_survey_by_user', user.id!);
+        gaEvent('submit_survey_response', params.surveyCode!, survey?.name);
```

Comment:
I'm not sure we should change these, just because we already have been collecting GA data with these, and by changing the name of the event it will skew the data

Reply:
I was thinking that since it's fairly early days in the scheme of things for datatrak it's okay to change these? But I might just change it back :+1:

Reply:
Maybe just check with Georgia, as I know she uses these